### DISCUSSION TITLE
Bluetooth

### DISCUSSION THREAD

**Discussion by joluga1948** [BimmerGestalt/AAIdrive#786] [General]
Hola no puedo abrir el bluetooth dentro de la app Aiidrive,y no puedo escuchar música ,solo recibo sms o watssaps, por medio de l a App.Un saludo y ,gracias
joluga1948@ telefonica.net

**Comment by hufman** [comment]
What do you mean, "open the bluetooth"? AAIdrive doesn't manage any bluetooth, it relies on Android to send the music to the car, and it relies on MyBMW to connect the apps to the car. If you are successfully receiving Whatsapp messages through the app, then the Apps connection is successful. Are you perhaps running over USB? In which case, you must tell the car to listen to the Bluetooth music source.
In the Music tab, do you see any compatible apps? Try installing some of the [known compatible apps](https://github.com/BimmerGestalt/AAIdrive?tab=readme-ov-file#implemented-features) to test with.
Since you have granted message access, it should also show an app entry for any running music player.

**Comment by joluga1948** [comment]
En la páguna de música,AAIdrive,la primera opcion,es activar bluetooth y no se activa,está opaco

**Reply by hufman** [reply]
You should share a screenshot, the music tab doesn't have any Bluetooth buttons that I remember...

**Comment by joluga1948** [comment]
https://ibb.co/qB3NJpW I don't hear anything here, I have to go to multimedia, and there is no
cover

https://ibb.co/0hGFgr9, pI can explore, but without sound

https://ibb.co/r4XMbJK, In audioplayer, vehicle information appears

https://postimg.cc/mtqdVf9K, here in Multimedia is where you listen to it via Bluetooth, without covers

https://ibb.co/n1WBPVW, mapas perfecto

https://ibb.co/9t7fpNf, perfect screen mirroring, full screen

https://ibb.co/0jgwd3n, playing and I don't hear anything.

https://ibb.co/xGnTS0b, Spotify doesn't appear here

**Comment by joluga1948** [comment]
Hello, I forgot this photo, where it says "not supported via USB", let's see if I can't hear the music on AAIdrive, is this why?


https://ibb.co/xDw2PGs,

I can't listen to Spotify via Bluetooth with AIdrive, any trick? Something has happened, I was listening to it a while ago (via USB) because everything comes out with covers, except the voice, I have the Note 20 Ultra and I tried several versions of My Bmw , I only listen to it on Multimedia. This is the question I had asked in a forum, talking about this topic




<https://ibb.co/xDw2PGs,>

**Comment by hufman** [comment]
Your car requires the Apps connection over USB, and your phone is too new to play music to the car over USB. You must have the car listening to music over Bluetooth.
I see the screenshots showing that the car is trying to listen to music from the music apps over USB. This only should happen if you had adjusted the Advanced Settings to force the audio context over USB, please disable this setting.

**Comment by Esco54** [comment]
is it possible to browse through folders in Spotify with ID6?

**Reply by hufman** [reply]
Of course, it's one of the best features of AAIdrive. Check in the music tab to make sure it has the right permissions.

**Comment by joluga1948** [comment]
OK This is what you're telling me, the permits? Bluetooth comes out, in gray, connectable and hidden, I don't know what it is. SDS

**Reply by hufman** [reply]
Oh, the Bluetooth "app" in the AAIdrive Music tab? That is not related to playing music through the car.
Your car should see your phone connection and you should tell the car to listen to the phone's Bluetooth connection for music, just like you would without AAIdrive installed.
If the music stops when you click the Audioplayer app in the car, that means the car is trying to listen to the USB connection for music, which won't work. Make sure you don't have `Force Audio Focus Over USB` enabled:
![image](https://github.com/BimmerGestalt/AAIdrive/assets/1592375/33e34a14-9e57-4e7b-9cc2-0d9f8e4fbd27)

**Comment by joluga1948** [comment]
Ok tanks
   
  
  
   
    El 26/06/2024 4:54 Walter Huf ***@***.***> escribió:
   
   
    
   
   
    
   
   
   Oh, the Bluetooth "app" in the AAIdrive Music tab? That is not related to playing music through the car.Your car should see your phone connection and you should tell the car to listen to the phone's Bluetooth connection for music, just like you would without AAIdrive installed.If the music stops when you click the Audioplayer app in the car, that means the car is trying to listen to the USB connection for music, which won't work. Make sure you don't have Force Audio Focus Over USB enabled:image.png (view on web)
